src/fsharp/IlxGen.fs

```diff
@@ -214,6 +214,24 @@ type IlxGenOptions =
       alwaysCallVirt: bool 
     }
 
+type IlxMethodInfo =
+    {
```

Comment:
Do we really need this record?  It is only used to pass a package of arguments around, it is out of style with the rest of the compiler and would bloat a lot if applied everywhere.  I'd say a tuple will be sufficient?

If it isn't then to strongly type a package of args use this?  That's wht's done in `fsc.fs` for example

```fsharp
type IlxMethodInfoArgs = IlxMethodInfoArgs of  vspec: Val * mspec: ILMethodSpec * access: ILMemberAccess * ...
```

Reply:
Yea, we probably don't need a record. I made one because I wanted `delayedGenMethods: Queue<IlxMethodInfo>` but I ended up going back to `delayedGenMethods: Queue<cenv -> unit>`. At that point it made sense to have the record, but now it probably doesn't and a tuple is fine.

Sometimes I struggle with when to choose a record over a single case union.

Reply:
Ok, it's done :)